code/modules/mob/living/simple_animal/abnormality/he/nameless_fetus.dm

```diff
@@ -0,0 +1,104 @@
+/mob/living/simple_animal/hostile/abnormality/fetus
+	name = "Nameless Fetus"
+	desc = "A giant, pus-filled baby."
+	icon = 'ModularTegustation/Teguicons/48x48.dmi'
+	icon_state = "fetus"
+	maxHealth = 400
+	health = 400
+	threat_level = HE_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(40, 50, 60, 60, 60),
+		ABNORMALITY_WORK_INSIGHT = list(20, 30, 30, 30, 30),
+		ABNORMALITY_WORK_ATTACHMENT = list(20, 30, 30, 30, 30),
+		ABNORMALITY_WORK_REPRESSION = list(20, 30, 30, 30, 30)
+		)
+	start_qliphoth = 1
+	pixel_x = -8
+	base_pixel_x = -8
+
+	max_boxes = 16
+	work_damage_amount = 7
+	work_damage_type = RED_DAMAGE
+	ego_list = list(
+//		/datum/ego_datum/weapon/syrinx,		Will update with the rest of the dump EGO
+		/datum/ego_datum/armor/syrinx
+		)
+//	gift_type =  /datum/ego_gifts/syrinx
+	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
+	var/mob/living/carbon/human/calling = null
+
+/mob/living/simple_animal/hostile/abnormality/fetus/ZeroQliphoth(mob/living/carbon/human/user)
+	check_players()
+	check_range()
+
+	//Are they nearby?
+/mob/living/simple_animal/hostile/abnormality/fetus/proc/check_range()
+	if(calling && (calling in range(1, src)))
```

Comment:
```suggestion
	if(in_range(calling, src))
```
`range` is a relatively expensive proc and should generally not be used in conditionals like this, especially when alternatives like checking `in_range(A, B)` or `A.Adjacent(B)` exist.